File: motor3d 2.0/objetogeometrico2d.vb
Namespace Espacio2D
    Public MustInherit Class ObjetoGeometrico2D
        Inherits Motor3D.ObjetoGeometrico

        Public Shadows Event Modificado(ByRef Sender As ObjetoGeometrico2D)

        Public Sub New()
            MyBase.New()
        End Sub

        Public Overrides Function ToString() As String
            Return MyBase.ToString()
        End Function
    End Class
End Namespace

File: motor3d 2.0.ConvertedToC#/ObjetoGeometrico2D.cs
using Microsoft.VisualBasic;
using System;
using System.Collections;
using System.Collections.Generic;
using System.Data;
using System.Diagnostics;
using System.Linq;
using System.Xml.Linq;
namespace Motor3D.Espacio2D
{
	public abstract class ObjetoGeometrico2D : Motor3D.ObjetoGeometrico
	{

		public new event ModificadoEventHandler Modificado;
		public new delegate void ModificadoEventHandler(ref ObjetoGeometrico2D Sender);

		public ObjetoGeometrico2D() : base()
		{
		}

		public override string ToString()
		{
			return base.ToString();
		}
	}
}
